addons/web/static/src/core/assets.js

```diff
@@ -108,23 +119,58 @@ export const loadBundleTemplates = memoize(async function loadBundleTemplates(na
     return processTemplates(templates);
 });
 
+
+/**
+ * Loads the content definition of a bundle
+ * @param {string} name the name of the bundle as declared in the manifest.
+ * @returns {Promise<BundleInfo[]>} A promise of the content definition of the bundle
+ * @private
+ */
+const _loadBundle = memoize(async function (bundleName) {
```

Comment:
Since this is not a method, `@private` doesn't really make sense. The function is private because it's not exported.

Also I think it should be renamed to something like `loadBundleInfo` to avoid confusion with the main `loadBundle` function.